python/pylibcudf/pylibcudf/column.pyx

```diff
@@ -135,7 +135,7 @@ cdef class OwnerMaskWithCAI:
         obj.owner = owner
 
         obj.cai = {
-            "shape": (bitmask_allocation_size_bytes(cv.size()),),
+            "shape": (bitmask_allocation_size_bytes(cv.size(), 64),),
```

Comment:
It appears `bitmask_allocation_size_bytes` only takes 1 argument?

https://github.com/rapidsai/cudf/blob/a26746d85e7384ed9dee9773e79ce5a7e5b2ede1/python/pylibcudf/pylibcudf/null_mask.pyx#L58